scripting/pugsetup/mapvote.sp

```diff
@@ -1,9 +1,52 @@
 #define RANDOM_MAP_VOTE "-1"  // must be in invalid index for array indexing
 
+char mapGroups[][] = {
+      "competitive.txt", "fun.txt", "maps.txt"
+};
+
 /**
  * Map voting functions
  */
 public void CreateMapVote() {
+          StartMappoolVote();
+}
+static void StartMappoolVote() {
+  Menu menu = new Menu(MappoolVoteHandler);
+  menu.SetTitle("%T", "VoteMenuTitle", LANG_SERVER);
+  menu.ExitButton = false;
+  for (int i = 0; i < sizeof(mapGroups); i++) {
+        char text[64], id[4];
+        Format(text, 64, mapGroups[i]);
+        Format(id, 4, "%i", i);
+        ReplaceString(text, 64, ".txt", "", false);
+        AddMenuItem(menu, id, text);
+  }
+  VoteMenuToAll(menu, g_MapVoteTimeCvar.IntValue);
+}
+public int MappoolVoteHandler(Menu menu, MenuAction action, int param1, int param2) {
+  if (action == MenuAction_VoteEnd) {
+        int winner = GetMenuInt(menu, param1);
+    
+        ServerCommand("sm_pugsetup_maplist %s", mapGroups[winner]);
```

Comment:
It doesn't make sense to override the value here. It  doesn't make sense to even have the cvar if the plugin will just override it before it gets used.